packages/draft-js-export-html/src/stateToHTML.js

```diff
@@ -36,9 +37,11 @@ type StyleMap = {[styleName: string]: RenderConfig};
 
 type BlockStyleFn = (block: ContentBlock) => ?RenderConfig;
 type EntityStyleFn = (entity: Entity) => ?RenderConfig;
+type InlineStyleFn = (style: DraftInlineStyle) => ?Object;
```

Comment:
I think this would be more robust if we could return a `? RenderConfig` like the other two. That way the option exists to specify element, attributes and/or style.

Reply:
agree, it'll be consistent with other methods